hack/verify-golint.sh

```diff
@@ -94,18 +100,29 @@ fi
 
 # check to make sure all packages that pass lint are in the linted file.
 echo
-if [ ${#linted[@]} -eq 0 ]; then
+if [ ${#linted[@]} -eq 0 -a ${#gone[@]} -eq 0 ]; then
```

Comment:
You're running `#!/bin/bash` -- always use `[[` over `[`. If using `[[`, use `&&` and `||` for logic. If you must be POSIX compliant, use `[` and `-a` and/or `-o`. Access to empty arrays with `-o nounset` is tricky but `${#array[@]}` is safe.